### ISSUE TITLE
iOS error: GoogleCast/GoogleCast.h file not found

### ISSUE THREAD

**Issue by saeedbashir** [googlecast/CastVideos-ios#69]
I'm trying to integrate Google Chrome Cast into my project. I tried to integrate it with cocoa pods by following the steps given [here](https://developers.google.com/cast/docs/ios_sender/).

My Project is Hybrid which means it has both Objective-C and Swift files. I'm able to import **GoogleCast/GoogleCast.h** in objective-C `.m` files but won't be able to import it in Swift files.

On importing GoogleCast module in Swift files it gives an error
`Could not build Objective-C module 'GoogleCast'`

To fix this issue I tried to add `GoogleCast/GoogleCast.h` in **edX-Bridging-Header.h** file but on doing this it starts giving me the error 
`GoogleCast/GoogleCast.h file not found`

**Note:** Xcode gave me the same error `GoogleCast/GoogleCast.h file not found` if I tried to import it in any `.h` file.

**Comment by saeedbashir** [comment]
@jtromo @helloyako Please help me in fixing this issue.

**Comment by helloyako** [comment]
@saeedbashir 
Did you add `import GoogleCast` in Swift file?

**Comment by saeedbashir** [comment]
> Did you add import GoogleCast in Swift file?

Yes, I did import it in the Swift file. On import, it gave me the error `Could not build Objective-C module 'GoogleCast'`
